proof/crefine/ARM/Finalise_C.thy

```diff
@@ -1262,8 +1260,7 @@ lemma deleteASID_ccorres:
                            asid_low_bits_def order_le_less_trans [OF word_and_le1])
     apply wp
    apply vcg
-  apply (clarsimp simp: Collect_const_mem if_1_0_0
```

Comment:
is Collect_const_mem now in the simpset?

Reply:
No, still unchanged, it just happened to not actually be used here (was probably a copy/paste). The change I was after was probably the `if_1_0_0`, because that generates a warning.